ng-file-upload/index.d.ts

```diff
@@ -41,7 +41,11 @@ declare module 'angular' {
              * Validate error name: minDuration
              * @type {(number|string)}
              */
-            ngfMinDuration: number | string;
+            ngfMinDuration?: number | string;
+            /**
+             * Validate error name: minSize
+             * @type {(number|string)}
```

Comment:
I don't know, it may be useless. I added it so that it is consistent with the other comments that also have a **@type**.